sunset/code/World.dm

```diff
@@ -0,0 +1,14 @@
+//This file is just for the necessary /world definition
+//Try looking in game/world.dm
+
+/world
+	mob = /mob/dead/new_player
+	turf = /turf/open/space/basic
+	area = /area/space
+	view = "15x15"
+	hub = "Exadv1.spacestation13"
+	name = "/Sunset/ Station 13"
+	fps = 60

```

Comment:
this isn't how you do it.

See https://github.com/OracleStation/OracleStation/pull/36